CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ For information on changes in released versions of Teku, see the [releases page]
 ## Unreleased Changes
 
 ### Additions and Improvements
-- Early access: Support for automatic fail-over of eth1-endpoints.  Multiple endpoints can be specified with the new `--eth1-endpoints` CLI option. Thanks to Enrico Del Fante. 
+- Early access: Support for automatic fail-over of eth1-endpoints.  Multiple endpoints can be specified with the new `--eth1-endpoints` CLI option. Thanks to Enrico Del Fante.
+- implement standard rest api `/eth/v2/beacon/blocks/:block_id` which supports altair blocks.
```

Comment:
nit:
```suggestion
- Implement standard rest api `/eth/v2/beacon/blocks/:block_id` which supports altair blocks.
```

Reply:
Actually might be worth adding a note in the changelog that the v2 APIs are not yet considered stable.